cargo-dist/tests/snapshots/akaikatana_basic.snap

```diff
@@ -185,10 +186,11 @@ download_binary_and_run_installer() {
             _updater_bin=""
             ;;
         *)
-            err "there isn't a package for $_arch"
+            err "internal installer error: $_artifact_name doesn't exist!?"
```

Comment:
Right, I guess that's because if we'd failed to pick a real archive in `select_archive_for_arch` we would have errored out before we got here?

Reply:
Yep